app/assets/stylesheets/content/_forms.sass

```diff
@@ -552,3 +552,11 @@ fieldset.form--fieldset
 
   li, div
     padding: 0
+
+// change password form
+form.change_password
+  max-width: 31em
+  min-height: 31em
```

Comment:
While I do admire your sense of accuracy, I would advise to see the visuals more as a guideline as your solution for a component should be more flexible. What if the form is longer? what if contains multiple columns of input elements?